specification/overview.md

```diff
@@ -26,21 +26,57 @@ This specification is designed to be used by three major groups:
+* The specification should provide all of the data elements necessary for the FinOps capabilities.
+
+### Provider-Neutral Approach by Default
+
+* While the schema, naming, terminology, and attributes of many providers are reviewed during development, this specification aims to be provider-neutral.
+* Contributors must take care to ensure the specification examines how each decision relates to each of the major cloud providers and SaaS vendors, not favoring any one.
+* In some cases, the approach may closely resemble one or more provider's implementation, while in other cases, the approach might be new. In all cases, the FOCUS group (community composed of FinOps practitioners, Cloud and SaaS providers and FinOps vendors) will attempt to prioritize alignment with the FinOps [Framework][FODOF] and [Capabilities][FODOFC].
 
 ### Extensibility
 
-The initial specification aims to introduce a common schema and terminology for billing datasets produced by Cloud Service Providers (CSPs). The specification however aims to be extensible to SaaS products and other types of cost datasets. Future versions of the specification will look to expand the content to support a broader set of prioritized FinOps capabilities.
+* The initial specification aims to introduce a common schema and terminology for billing datasets produced by Cloud Service Providers (CSPs).
+* The specification however aims to be extensible to SaaS products and other types of cost datasets.
+* Future versions of the specification will look to expand the content to support a broader set of prioritized FinOps capabilities.
+
+## Design Notes
+
+### Optimize for data analysis
+
+* Columns should be optimized for data analysis (aggregations and breakdowns) at scale and should not require splitting or parsing.
+* Data that cannot be represented in predefined columns (e.g., tags) should use a simplified JSON object that does not use nested objects and arrays, and is parseable within spreadsheets.
+* Avoid complex JSON structures for any scenario that requires non-technical personas or has a well-defined structure that could be represented as separate, flat columns.
+* FOCUS should include all data necessary as the system of record for cost and usage data and scenarios that require this dataset.
+* Minimize the need for expensive joins at scale for non-technical personas and avoid duplicating data for scenarios that require engineering support where complex joins are more feasible (e.g., forecasting).
+
+### Consistency helps with clarity
+
+* Where possible use consistent names that will naturally group like data in the specification
+* Use common patterns to reinforce learning, (e.g. Id dimensions always contain an identifier, Category dimensions are always normalized)
```

Comment:
Hmm... We should define this in the column naming attribute.
```suggestion
* Use common patterns to reinforce learning (e.g., ID columns always contain an identifier, Category columns are always normalized).
```

Reply:
That sounds like the right approach. I will look at that section